buildrules/linker.rules

```diff
@@ -25,3 +25,5 @@ run.x86.debug.gdb: $(OBJDIR)/boot.img
 	qemu-system-x86_64 -cpu 486 -fda $< -net none -no-kvm -S -s -monitor stdio
 run.x86.libvirt: $(OBJDIR)/boot.img
 	virt-install --name kos --ram 64 --disk $<,device=floppy,format=raw --boot hd --arch i386
+run.vnc: $(OBJDIR)/boot.img
+	qemu-system-x86_64 -d cpu_reset -cpu 486 -fda $< -net none -no-kvm -monitor stdio -serial pty -vnc 0.0.0.0:9 -k en-us
```

Comment:
Please give an explanation for this line.

Reply:
I mean more explain is better, for example, '-d cpu_reset' seems like a new parameter.